test/integration/roles/test_git/tasks/main.yml

```diff
@@ -495,7 +551,7 @@
 
 - assert:
     that: "not checkout2|changed"
-  when: git_version.stdout | version_compare("{{git_version_supporting_depth}}", '>=')
+  when: git_version.stdout | version_compare("{{git_version_supporting_get_url}}", '>=')
```

Comment:
The tests that check for the version fail without the version check. Versions lower than 2.7 will report being changed due to not having `git remote get-url`.